BE/db/index.js

```diff
@@ -306,6 +327,41 @@ async function getUserPageRecipeById(recipeId){
 
 }
 
+  //GET OTHER PAGE RECIPE BY ID
+  async function getOtherPageRecipeById(recipeId){
```

Comment:
What is this used for?

Reply:
That is the recipe list data that includes userInfo, for getAllRecipesPage and getReviewedRecipesPage